roles/solrcloud/tasks/config.yml

```diff
@@ -1,71 +1,58 @@
 ---
+
 - name: Solrcloud | Check for solr znode
   ansible.builtin.command: >
     {{ solr_installation }}/bin/solr zk ls /
-    -z {{ solr_zookeeper_hosts_string }}
+    -z {{ zookeeper_hosts | join(',') }}
   register: znode_check
   ignore_errors: true
   changed_when: false
 
 - name: Solrcloud | Create zk chroot / solr znode using bin/solr
   ansible.builtin.command: >
     {{ solr_installation }}/bin/solr zk mkroot {{ solr_znode }}
-    -z {{ solr_zookeeper_hosts_string }}
+    -z {{ zookeeper_hosts | join(',') }}
   when: solr_znode not in znode_check.stdout_lines and solr_znode != ''
   changed_when: false
 
 - name: Solrcloud | Configure SolrCloud init script
   ansible.builtin.template:
     src: 'solr.in.sh.j2'
-    dest: '/etc/default/solr.in.sh'
+    dest: '/opt/solr/bin/solr.in.sh'
     owner: '{{ solr_user }}'
     group: '{{ solr_group }}'
     mode: "0644"
     force: true
   changed_when: false
-  notify: restart SolrCloud
+  notify: Restart solr
 
 - name: Solrcloud | Configure high ulimit value for Solr
   ansible.builtin.template:
     src: 'solr.conf.j2'
     dest: '/etc/security/limits.d/solr.conf'
     mode: "0644"
     force: true
-  notify: restart SolrCloud
+  notify: Restart solr
 
 - name: Solrcloud | Configure SolrCloud properties
   ansible.builtin.template:
-    src: 'solr.xml.j2'
-    dest: '{{ solr_data_dir }}/solr.xml'
-    owner: '{{ solr_user }}'
-    group: '{{ solr_group }}'
-    mode: "0644"
-    force: true
-  notify: restart SolrCloud
-  when:
-    - '"8" in solr_cloud_download_version'
-
-- name: Solrcloud | Configure SolrCloud 9 properties
-  ansible.builtin.template:
-    src: 'solr9.xml.j2'
+    src: solr.xml.j2
     dest: '{{ solr_data_dir }}/solr.xml'
     owner: '{{ solr_user }}'
     group: '{{ solr_group }}'
     mode: "0644"
     force: true
-  notify: restart SolrCloud
-  when:
-    - '"9" in solr_cloud_download_version'
+  notify: Restart solr
 
-- name: solrcloud | Configure pre 7.4 logging
+- name: Solrcloud | Configure pre 7.4 logging
```

Comment:
It's odd that this task name references 7.4 - is that a Java version? a Solr version? Have we moved beyond the need for this?